packages/elements/src/react/sign-in/identifiers.tsx

```diff
@@ -0,0 +1,19 @@
+import {
+  SignInNameSelector,
+  SignInSafeIdentifierSelector,
+  SignInSalutationSelector,
+} from '~/internals/machines/sign-in/selectors/sign-in.selectors';
+
+import { SignInRouterCtx } from './context';
+
+export function SignInSafeIdentifier(): string {
+  return SignInRouterCtx.useSelector(SignInSafeIdentifierSelector);
+}
+
+export function SignInSalutation(): string {
+  return SignInRouterCtx.useSelector(SignInSalutationSelector);
+}
+
+export function SignInName(): string {
+  return SignInRouterCtx.useSelector(SignInNameSelector);
+}
```

Comment:
I'm wondering if we wrap these in a `<span>` to ensure we're consistently rendering markup from our components. The alternative if someone wants to style these now is to wrap them in an element, which I would say isn't consistent with the rest of the component surface at the moment (I think we're always rendering markup for components that render something).

Reply:
Not always (ex: `SignIn`). That said, I'm torn. 

On one hand, I agree. 
On another, is that really a problem? 

Meaning do we want to surface more code and force them to have additionaly elements around something they'll likely wrap in an alternate way (similarly to how I've used them in the example sign-in page)?

Reply:
It's not a problem really, just more of a consistency thing and being predictable in how our components behave. We can always adjust down the road though, so happy to ship with just text here.